data/libs/Commodities.lua

```diff
@@ -0,0 +1,63 @@
+-- Copyright © 2008-2020 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+local Economy = require 'Economy'
+
+local commodities = Economy.GetCommodities()
+local economies = Economy.GetEconomies()
+
+local EquipType = _G.EquipType
```

Comment:
Why is it necessary to use a global table here instead of simply requiring EquipType ?

Reply:
There's a dependency hell going on here where Commodity.lua needs access to EquipType, but the rest of Equipment.lua needs access to the commodity data, because everything is interlinked. The global assignment is just a quick hack - the solution is to add a "preregistration" mechanism to the `package` table so that circular dependencies aren't a problem. I've built one of those before and it makes a *huge* difference for complex lua projects.

I might merge the contents of Commodity.lua back into Equipment.lua, but this was the first step in separating commodities from equipment so I kinda want to keep it that way.